### PR TITLE
Cater for digital nomads

### LINKED ISSUE #92: Consider adding "sync/async" information to the companies listing
On the BcnEng Slack, someone was asking about companies that hire on Spain, but are async enough so you could work as a digital nomad from a different country sporadically. Adding this information to the company listing would help people in this situation.

I feel making this information "mandatory" would be impractical- it would make vetting companies considerably harder, but surely we could request people to opt in to adding tags like "completely async", "UTC-2 - UTC+6 mandatory", etc.

### PR DESCRIPTION
**Pull request by alexpdp7**
Closes #92

### CODE REVIEW

**Review by uesteibar — APPROVED** [review]
Looks good to me, do you have any idea on how to get the existing records to fill in the information? maybe we can make a "call for updates" in the BCN engineering slack group?

**Comment by alexpdp7** [comment]
> Looks good to me, do you have any idea on how to get the existing records to fill in the information? maybe we can make a "call for updates" in the BCN engineering slack group?

The discussion on digital nomads in BcnEng contained some interesting things I didn't know: mostly that working as a regular employee in Spain from abroad is... problematic:

https://bcneng.slack.com/archives/C42GY1WN5/p1660890441389739?thread_ts=1660752356.000229&cid=C42GY1WN5

; such as not getting paid if you get sick (!?). So I'm thinking of backtracking here a bit and changing "work for extended periods of time abroad" to "work *briefly* abroad", because it seems that working for extended periods outside is an idea I don't want to "endorse".

This might seem a bit unrelated to your question, but I think while this information (sync/async) is useful, we shouldn't make a huge effort to standardize it (it is not essential to our target users)- we should just guide people writing offers to include this information in a standardized manner if they include it.

**Comment by uesteibar** [comment]
@alexpdp7 there are legal ways to work from outside your country for quite some time, legally, and with social security coverage. Only that the company needs to be willing to back you up.

However, you do have a point here. How about we don't mention "working abroad", but instead "working on a different timezone or schedule"?

**Comment by alexpdp7** [comment]
@uesteibar see https://github.com/remote-es/remotes/pull/93/commits/f9f7f7a340b96bdea708c15189eb511be6e4c54a .

File: README.md [hunk 1]
```diff
@@ -82,6 +80,12 @@ Tag your offer with:
   not offer any indication that all offers are remote, nor marks clearly
   remote offers, so you need to check individual offers to see if the
   position is remote.
+* (Async friendly)/(Async first) if you do not have specific availability
+  requirements. This is especially interesting for workers who might be
+  interested in working on a different scheduoe or timezone. You can also
```
Comment (uesteibar) [inline README.md:85]:
```suggestion
  interested in working on a different schedule or timezone. You can also
```

**Pull request merged**